raiden-dapp/src/components/TokenOverlay.vue

```diff
@@ -127,6 +127,7 @@ export default class TokenOverlay extends Mixins(
     if (token === tokenAddress) {
       this.cancel();
     } else {
+      this.cancel();
```

Comment:
If `cancel()` needs to be called in every situation we can just pull it ouside and change the if to `token !== tokenAddress` to call `this.navigateToSelectTransferTarget(tokenAddress)`

Reply:
Really nice suggestion :)

Reply:
Alrighty, will do!